src/Sylius/Bundle/ApiBundle/Resources/config/routing/shipping_category.yml

```diff
@@ -7,6 +7,7 @@ sylius_api_shipping_category_index:
     defaults:
         _controller: sylius.controller.shipping_category:indexAction
         _sylius:
+            serialization_groups: [Default]
```

Comment:
Right now:

https://github.com/Sylius/SyliusResourceBundle/blob/master/Controller/Configuration.php#L369

And I think JMS requires to specify Default explicitly as it won't add it automatically if you pass an empty array - not sure. So I think we should add 'Default' to the Configuration::getSerializationGroups() method which I linked above.